plasmapy/mathematics/tests/test_dispersion.py

```diff
@@ -1,92 +1,167 @@
 """Tests for the plasma dispersion function and its derivative"""
 
 import numpy as np
+from numpy import pi as π
```

Comment:
I've never seen this! 🍰

Reply:
And sooner or later this is going to break something, I guarantee it. Nice syntactic sugar but in all honesty, let's just keep to good old fashioned ASCII English in code. Please, say no to symbol voodoo.